roles/uc-ui/templates/config.yml.j2

```diff
@@ -0,0 +1,85 @@
+# This file should not be modified as it may get changed by the maintainer.
+# Add extra configuration files in the extra_config_files directory to override
+# the content from this file
+
+# Extra configuration directory where the config files overriding this file
+# should be.
+extra_config_files: /etc/wazo-ui/conf.d/
+
+# Change the log level to debug level.
+debug: false
+
+# Log file.
+log_filename: /var/log/wazo-ui.log
+
+# PID file.
+pid_filename: /var/run/wazo-ui/wazo-ui.pid
+
+https:
+  listen: {{ uc_ui_listen_address }}
+  port: {{ uc_ui_port}}
+  certificate: /usr/share/xivo-certs/server.crt
+  private_key: /usr/share/xivo-certs/server.key
```

Comment:
Since it's the same value than the default, remove it

Reply:
ok